pkg/transport/process/extensions/extensions_suite_test.go

```diff
@@ -0,0 +1,106 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+package extensions_test
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/gardener/component-cli/pkg/transport/process"
+	"github.com/gardener/component-cli/pkg/transport/process/extensions"
+	cdv2 "github.com/gardener/component-spec/bindings-go/apis/v2"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	defaultProcessorBinaryPath = "../../../../tmp/test/bin/processor"
+)
+
+func TestConfig(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "transport extensions Test Suite")
+}
+
+var _ = Describe("transport extensions", func() {
```

Comment:
we should check if the defaultProcessor binary has been installed in a `BeforeSuite` and faile with a understandable error message